Adyen/UI/List/ListItem.swift

```diff
@@ -37,31 +62,35 @@ public class ListItem: FormItem {
     /// that is set to the `ListItemView.accessibilityIdentifier`.
     public var identifier: String?
 
-    /// The flag to indicate if an icon is a custom image that should not be tempered.
-    public let canModifyIcon: Bool
+    /// Specifies the icon behavior
+    public let iconMode: IconMode
```

Comment:
This is breaking change 😅
If you planning for this feature to be in v5 - need to deprecate the old one

Reply:
👌 Fixed here 6e9581df7f7ea068c1c60ff17f36787b5de9f3c1

Reply:
I don't think spi code requires deprecation. It's the purpose of the spi block. It's meant for our usage between modules. If change is needed, we can do it freely without worrying about public interface change, since this is not "public" interface.